cfg/process.py

```diff
@@ -286,9 +286,9 @@ def cfg_yielder(model, binary):
 		#-----------------------------------------------------
 		elif d['type'] == '[route]': # add new layer here
 			routes = d['layers']
-			if type(routes) is str:
-				routes = [int(x.strip()) for x in routes.split(',')]
-			else: routes = [routes]
+			if type(routes) is int:
```

Comment:
Agreed. But Python2 would potentially cause other types of error down the line. Hope you upgrade to Python3 soon.